CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## 2.2.4
```

Comment:
😬 this is gonna conflict with https://github.com/18F/identity-style-guide/pull/159/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed

Reply:
> 😬 this is gonna conflict with https://github.com/18F/identity-style-guide/pull/159/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed

Yeah, there's going to be a lot of rebase surgery to be done between all the open pull requests 😄